### DISCUSSION TITLE
Uninstall fails

### DISCUSSION THREAD

**Discussion by dermanu** [toltec-dev/toltec#753] [Support] [answered]
Hi there,

I try to uninstall toltec using `toltecctl uninstall`. However, I got the following message: 
```
This will wipe out all files in '/opt'. Continue? [y/N] y
wget: unrecognized option '--no-verbose'
BusyBox v1.35.0 () multi-call binary.

Usage: wget [-cqS] [--spider] [-O FILE] [-o LOGFILE] [--header STR]
	[--post-data STR | --post-file FILE] [-Y on/off]
	[-P DIR] [-U AGENT] [-T SEC] URL...
Unable to fetch standalone opkg, make sure you have a stable Wi-Fi connection
```

Wifi is connected. The OS version of my remarkable 2 is 3.8.1.1960 (Linux reMarkable 5.4.70-v1.3.4-rm11x #1 SMP PREEMPT Wed Mar 15 06:06:44 UTC 2023 armv7l GNU/Linux). 

It might be due to me forgetting that I had toltec installed (as I find its bootstraper and .entware) and then updating the Remarkable (even enrolling in the beta version... :sweat_smile:). I have not removed anything yet, like bootstrap or installing toltec again. I also can't find opkg on the device. Wanted to first ask how to proceed.

All the best, 
dermanu

**Comment by Eeems** [comment, marked as answer]
Toltec doesn't support 3.8, so it isn't enabled. You don't need to uninstall it. You can always downgrade to a supported version, re-enable and then uninstall if you really need to, otherwise you can just leave it in its not enabled state.

**Reply by dermanu** [reply]
Ah, I see. Smart feature. Can I also do a factory reset now without bricking it? And sorry for posting this question first into issues.

**Reply by Eeems** [reply]
https://remarkable.guide/tech/factory-reset.html

I would recommend doing a second system upgrade/downgrade before doing a factory reset. This way your fallback partition will not be in a broken state.

**Reply by dermanu** [reply]
Perfect! Thanks!
